tests-perf/locust/locust-notifications.py

```diff
@@ -26,15 +26,15 @@
 
 class NotifyApiUser(HttpUser):
 
-    wait_time = constant_pacing(60)
+    wait_time = constant_pacing(60)  # 60 seconds between each task
     host = os.getenv("PERF_TEST_DOMAIN", "https://api.staging.notification.cdssandbox.xyz")
 
     def __init__(self, *args, **kwargs):
         super(NotifyApiUser, self).__init__(*args, **kwargs)
 
         self.headers = {"Authorization": os.getenv("PERF_TEST_AUTH_HEADER")}
         self.email = os.getenv("PERF_TEST_EMAIL", "success@simulator.amazonses.com")
-        self.phone_number = os.getenv("PERF_TEST_PHONE_NUMBER", "16132532222")
+        self.phone_number = os.getenv("PERF_TEST_PHONE_NUMBER", "16135550123")
```

Comment:
we're not using this right now, but I changed to our internal test number just in case we do in the future.